lib/engine/ability/README.md

```diff
@@ -254,6 +256,10 @@ Modified station token placement
   effectively increasing the city's size by one. (See 18 Los Angeles's optional
   company "Dewey, Cheatham, and Howe" or the corporations which get removed in
   1846 2p Variant for examples). Default nil.
+- `bull`: Simlar to `cheater` except this token does not take a slot -
```

Comment:
can you rename bull to extra_slot or something

why did you name it bull?

Reply:
I named it bull because it made me think of the bull tokens in 18C2C where merger corps get a bull token they can place in tokened out cities this way. I'll rename it `extra_slot`